gateway/tests/api/test_v1_program.py

```diff
@@ -9,7 +9,7 @@
 from rest_framework import status
 from rest_framework.test import APITestCase
 
-from api.models import Job
+from api.models import Job, Program
```

Comment:
assuming since the gateway components aren't user-facing we don't need to rename this to Function :crossed_fingers:

Reply:
Correct, internally I'm not renaming models and so on. What I'm changing are errors messages or responses returned to the user. If you see any reference that I should change let me know 😄